tests/s3/test_object_tagging.py

```diff
@@ -984,10 +984,11 @@ def test_duplicate_name_object_tag_support_2477(self):
     @pytest.mark.s3_object_tags
     @pytest.mark.tags("TEST-5562")
     @CTFailOn(error_handler)
-    def test_max_object_max_tags_2478(self):
-        """Verification of max. no. of Objects user can upload with max no. of tags per Object."""
-        self.log.info(
-            "Verification of max. no. of Objects user can upload with max no. of tags per Object")
+    def test_multiple_object_max_tags_2478(self):
+        """Verification of multiple no. of Objects user
+        can upload with max no. of tags per Object."""
+        self.log.info("Verification of multiple no. of Objects "
+                      "user can upload with max no. of tags per Object")
         self.log.info(
```

Comment:
We follow line length limit up to 100 characters, can you please follow same to reduce number of lines from this test module?

Reply:
Addressed & updated.